Correct Manage disk space section (#7171)
magento/devdocs: src/cloud/project/manage-disk-space.md

> You can check disk space usage in each environment using the _disk free_ command, which reports the amount of disk space used by the file system. You must use SSH to log in to a remote environment.

Reviewer: For example, you could update this part with something about the `mount:size` command and sample response:

```suggestion
You can check disk space usage in the current environment using the `magento-cloud mount:size` command:
```

```bash
magento-cloud mount:size
```

Sample response:

```terminal
Checking disk usage for all mounts on <project>-<branch>-mymagento@ssh.us.magento.cloud...

+------------+-----------+---------+-----------+-----------+--------+
| Mount(s)   | Size(s)   | Disk    | Used      | Available | % Used |
+------------+-----------+---------+-----------+-----------+--------+
| app/etc    | 184 KiB   | 1.9 GiB | 481.3 MiB | 1.4 GiB   | 24.7%  |
| pub/media  | 128 KiB   |         |           |           |        |
| pub/static | 158.2 MiB |         |           |           |        |
| var        | 316.7 MiB |         |           |           |        |
+------------+-----------+---------+-----------+-----------+--------+

All the mounts share the same disk.

To increase the available space, edit the disk key in the .magento.app.yaml file.
```

```
Also, you can check disk space usage in the remote environments using the _disk free_ command, which reports a more detailed list of the amount of disk space used by the file system. You must use SSH to log in to a remote environment.
```

Author: **Staging and Production environments**: to check the disk space on these clusters "_df -h_" can be used. Sample output below. _/data/exports/_ mount shows the disk space for media and _/data/mysql/_ mount shows disk space for database 

```
sngst3u29h5gf@i-04422e3b24f2ef115:~$ df -h
Filesystem                                    Size  Used Avail Use% Mounted on
udev                                           16G     0   16G   0% /dev
tmpfs                                         3.2G  9.1M  3.2G   1% /run
/dev/xvda1                                     59G  8.9G   48G  16% /
tmpfs                                          16G   36K   16G   1% /dev/shm
tmpfs                                         5.0M     0  5.0M   0% /run/lock
tmpfs                                          16G     0   16G   0% /sys/fs/cgroup
/dev/xvdj                                     9.8G  2.3G  7.6G  23% /data/mysql
/dev/xvdi                                     9.8G  491M  9.3G   5% /data/exports
192.168.5.5:/shared                           9.8G  591M  9.3G   6% /mnt/shared
/dev/loop0                                     91M   91M     0 100% /app/sngst3u29h5gf
192.168.5.5:/shared/sngst3u29h5gf/var         9.8G  591M  9.3G   6% /app/sngst3u29h5gf/var
192.168.5.5:/shared/sngst3u29h5gf/app/etc     9.8G  591M  9.3G   6% /app/sngst3u29h5gf/app/etc
192.168.5.5:/shared/sngst3u29h5gf/pub/media   9.8G  591M  9.3G   6% /app/sngst3u29h5gf/pub/media
192.168.5.5:/shared/sngst3u29h5gf/pub/static  9.8G  591M  9.3G   6% /app/sngst3u29h5gf/pub/static
```
**Integration environment and Starter projects**: Use "_magento-cloud mount:size_"

Sample response:

```
Checking disk usage for all mounts on <project>-<branch>-mymagento@ssh.us.magento.cloud...

+------------+-----------+---------+-----------+-----------+--------+
| Mount(s)   | Size(s)   | Disk    | Used      | Available | % Used |
+------------+-----------+---------+-----------+-----------+--------+
| app/etc    | 184 KiB   | 1.9 GiB | 481.3 MiB | 1.4 GiB   | 24.7%  |
| pub/media  | 128 KiB   |         |           |           |        |
| pub/static | 158.2 MiB |         |           |           |        |
| var        | 316.7 MiB |         |           |           |        |
+------------+-----------+---------+-----------+-----------+--------+
```

Reviewer: @SumanPitta  You need to update **your** PR with the changes you are proposing. Otherwise, you can bring this information up as a GitHub issue instead, and we will take the content of the GitHub issue under advisement and apply the changes accordingly.

Author: @hguthrie seems update could not be made. You may take the content and update the devdocs if necessary.

Reviewer: Okay @SumanPitta  Thanks for the updates. I will get this sorted out. 👍

{:.bs-callout-info}

> The _df -h_ command is only for dedicated clusters i.e. use it on Staging and Production environments of Magento Cloud Pro projects. On the Integration environments and for Magento Cloud Starter projects, use _magento-cloud db:size_

Reviewer 2: ```suggestion
The  _df -h_ command works only on {{site.data.var.ece}} Pro Production and dedicated Staging environments. To check disk space on Integration environments and Magento Cloud Starter environments,  use the `magento-cloud db:size` command.
```

> The _df -h_ command is only for dedicated clusters i.e. use it on Staging and Production environments of Magento Cloud Pro projects. On the Integration environments and for Magento Cloud Starter projects, use _magento-cloud db:size_

Author: **Staging and Production environments**: to check the disk space on these clusters "df -h" can be used. Sample output below. /data/exports/ mount shows the disk space for media and /data/mysql/ mount shows disk space for database

```
sngst3u29h5gf@i-04422e3b24f2ef115:~$ df -h
Filesystem                                    Size  Used Avail Use% Mounted on
udev                                           16G     0   16G   0% /dev
tmpfs                                         3.2G  9.1M  3.2G   1% /run
/dev/xvda1                                     59G  8.9G   48G  16% /
tmpfs                                          16G   36K   16G   1% /dev/shm
tmpfs                                         5.0M     0  5.0M   0% /run/lock
tmpfs                                          16G     0   16G   0% /sys/fs/cgroup
/dev/xvdj                                     9.8G  2.3G  7.6G  23% /data/mysql
/dev/xvdi                                     9.8G  491M  9.3G   5% /data/exports
192.168.5.5:/shared                           9.8G  591M  9.3G   6% /mnt/shared
/dev/loop0                                     91M   91M     0 100% /app/sngst3u29h5gf
192.168.5.5:/shared/sngst3u29h5gf/var         9.8G  591M  9.3G   6% /app/sngst3u29h5gf/var
192.168.5.5:/shared/sngst3u29h5gf/app/etc     9.8G  591M  9.3G   6% /app/sngst3u29h5gf/app/etc
192.168.5.5:/shared/sngst3u29h5gf/pub/media   9.8G  591M  9.3G   6% /app/sngst3u29h5gf/pub/media
192.168.5.5:/shared/sngst3u29h5gf/pub/static  9.8G  591M  9.3G   6% /app/sngst3u29h5gf/pub/static
```
**Integration environment and Starter projects**: Use "magento-cloud mount:size"

Sample response:

```
Checking disk usage for all mounts on <project>-<branch>-mymagento@ssh.us.magento.cloud...

+------------+-----------+---------+-----------+-----------+--------+
| Mount(s)   | Size(s)   | Disk    | Used      | Available | % Used |
+------------+-----------+---------+-----------+-----------+--------+
| app/etc    | 184 KiB   | 1.9 GiB | 481.3 MiB | 1.4 GiB   | 24.7%  |
| pub/media  | 128 KiB   |         |           |           |        |
| pub/static | 158.2 MiB |         |           |           |        |
| var        | 316.7 MiB |         |           |           |        |
+------------+-----------+---------+-----------+-----------+--------+
```
